io/src/main/scala/fs2/io/file/Files.scala

```diff
@@ -151,9 +152,10 @@ sealed trait SyncFiles[F[_]] {
   /** Creates a resource containing the path of a temporary directory.
     *
     * The temporary directory is removed during the resource release.
```

Comment:
You didn't write this, but I think it can be improved, I found the wording slightly confusing. How about something like:

> Creates a temporary directory, and returns its path. The directory is removed during the release of the resulting `Resource`

Same applies to the `tempFile` scaladoc above

Reply:
Reworded + added some bits from nio Files' javadoc.